block_element.go

```diff
@@ -244,7 +244,7 @@ func NewDatePickerBlockElement(actionID string) *DatePickerBlockElement {
 // More Information: https://api.slack.com/reference/messaging/block-elements#input
 type PlainTextInputBlockElement struct {
 	Type         MessageElementType `json:"type"`
-	ActionID     string             `json:"action_id"`
+	ActionID     string             `json:"action_id,omitempty"`
```

Comment:
According to https://api.slack.com/reference/block-kit/block-elements#input, action_id is required.

Reply:
@samolds did you see @kanata2's comment? ^

Reply:
oops, i missed this. thanks for looking out!